packages/engine/src/scene/components/PostProcessingComponent.tsx

```diff
@@ -62,52 +57,17 @@ export const PostProcessingComponent = defineComponent({
 
 function PostProcessingComponentReactor(): ReactElement {
   const entity = useEntityContext()
-  const component = useComponent(entity, PostProcessingComponent)
+  const component = useHookstate(useComponent(entity, PostProcessingComponent))
 
   useEffect(() => {
     getMutableState(PostProcessingSettingsState).enabled.set(component.enabled.value)
   }, [component.enabled])
 
-  if (!EngineRenderer.instance?.effectComposer) return <></>
-
-  return (
-    <>
-      {Object.entries(getComponent(entity, PostProcessingComponent).effects)
-        .map(([name, effect]) =>
-          Object.entries(effect).map(([key, value]) => (
-            <PostProcessingEffectPropertyReactor
-              effectState={component.effects[name]}
-              effectName={name}
-              property={key}
-              key={name + key}
-              value={value}
-            />
-          ))
-        )
-        .flat()}
-    </>
-  )
-}
+  useEffect(() => {
+    getMutableState(PostProcessingSettingsState).effects.set(component.effects.get(NO_PROXY_STEALTH))
```

Comment:
similar to above, we need a separate reactor for each effect, and do a merge, or we may be losing data